src/source/src/connector_source.rs

```diff
@@ -263,7 +263,7 @@ impl ConnectorSource {
         splits: ConnectorState,
         column_ids: Vec<ColumnId>,
     ) -> Result<ConnectorSourceReader> {
-        let (tx, rx) = mpsc::unbounded_channel();
+        let (tx, rx) = mpsc::channel(CONNECTOR_MESSAGE_BUFFER_SIZE);
```

Comment:
Would it be better to set the number of splits instead of 64

Reply:
Is there any reason behind that?

Reply:
64 should be okay, just ignore me

Reply:
It's too small 🤤🤤 still benchmarking. I'm guessing it should be 16 * chunk size.